### ISSUE TITLE
Dialog aria-labelledby references an unexisting id

### ISSUE THREAD

**Issue by sylven32** [primefaces/primereact#1405]
It's set with aria-labelledby={this.id + '\_label'}, see line:
https://github.com/primefaces/primereact/blob/9133fef966d6a644b67dea5935f20ef9cbfc4896/src/components/dialog/Dialog.js#L389
It should be aria-labelledby={this.id + '\_**header**'}

**Comment by sylven32** [comment]
Any ETA on this? Thanks in advance for your help.
